databaseserviceexample.gitlab-ci.yml

```diff
@@ -0,0 +1,45 @@
+include: 
+  - https://raw.githubusercontent.com/Cimpress-MCP/gitlab-continuous-integration/master/belt-node-complexity-analysis-ci.yml
+  - https://raw.githubusercontent.com/Cimpress-MCP/gitlab-continuous-integration/master/deploy/deploycomponents.yaml
+  - https://raw.githubusercontent.com/Cimpress-MCP/gitlab-continuous-integration/master/deploy/beanstalkdeploy.yaml
+  - https://raw.githubusercontent.com/Cimpress-MCP/gitlab-continuous-integration/master/setup/nodesetup.yaml
+  - https://raw.githubusercontent.com/Cimpress-MCP/gitlab-continuous-integration/master/service/dependencyscanning.yaml
+  - https://raw.githubusercontent.com/Cimpress-MCP/gitlab-continuous-integration/master/databasae/builddatabase.yaml
+  - https://raw.githubusercontent.com/Cimpress-MCP/gitlab-continuous-integration/master/databasae/stages.yaml
+
+build:myDataBaseService:
+  extends: .buildDatabaseWithNpm
+  variables:
+    POSTGRES_VERSION: postgres:10.6-alpine
+    POSTGRES_DB: postgres
+    POSTGRES_USER: postgres
+    POSTGRES_SERVER: postgres
+    POSTGRES_PASSWORD: ''
+
+migrate-int:
+  extends: .migrate
+  stage: migrate-int
+  only:
+    - master
+  except:
+    - schedules
+  variables:
+    POSTGRES_SERVER: myDatabaseServerAddress.com
+    POSTGRES_USER: postgres
```

Comment:
`postgres` isn't a good placeholder for this. It tends to be the default root user. We probably shouldn't be using that root user in our day-to-day tasks.

something more like "myDeployUser" or "myMigrateUser" or "myReleaseManagementUser"